lib/sidekiq/cli.rb

```diff
@@ -269,6 +269,19 @@ def validate!
         logger.info @parser
         die(1)
       end
+
+      errors = []
```

Comment:
It's only 2 parameters with 2 possible error conditions - fail fast works
for me. Patch updated.

On Fri, May 15, 2015 at 5:00 AM Anton Antonov <notifications@github.com>
wrote:

> In lib/sidekiq/cli.rb
> <https://github.com/mperham/sidekiq/pull/2352#discussion_r30394222>:
>
> > @@ -269,6 +269,19 @@ def validate!
> >          logger.info @parser
> >          die(1)
> >        end
> > +
> > +      errors = []
>
> In favor of collecting errors, wouldn't a setting as :fail_on_first
> (default - true) solve this problem and make both sides happy?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mperham/sidekiq/pull/2352/files#r30394222>.
>